### DISCUSSION TITLE
The "Difficulty Adjust" mod should be hidden if it does not change any beatmap value

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#19143] [Q&A] [answered]
As title. There may be a case where the user forgets to have the "difficulty adjust" mod enabled, and it may happen that the mod is not adjusting any value. So visualising it, in that case, is not needed:
![image](https://user-images.githubusercontent.com/43073074/179354108-94918e9e-f882-4229-b73f-0d1a22e8282b.png)

**Comment by Theighlin** [comment, marked as answer]
#17835
